Sources/Benchmark/BenchmarkReporter.swift

```diff
@@ -12,22 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import Dispatch
```

Comment:
It's better to use `BenchmarkClock` here, so that we can isolate dependency on Dispatch (and maybe remove it later in the future by implementing clock based on something more lightweight).

Reply:
Ack; SGTM!